extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -33,14 +36,37 @@ export class AmpMustache extends AMP.BaseTemplate {
 
   /** @override */
   compileCallback() {
+    /** @private @const {!JsonObject} */
+    this.nestedTemplates_ = dict();
+    const shadowTemplate = this.element.cloneNode(true);
+    let index = 0;
+    iterateCursor(templateContent(shadowTemplate).querySelectorAll('template'),
```

Comment:
What is this iterating? It seems you're making a clone, iterating it, then throwing away the entire result.

Reply:
Fixed